module/VuFind/src/VuFind/View/Helper/Root/Config.php

```diff
@@ -88,4 +88,20 @@ public function ajaxCoversEnabled()
     {
         return $this->get('config')->Content->ajaxcovers ?? false;
     }
+
+    /**
+     * Get advanced search limits that override the natural sorting to
+     * display at the top.
+     *
+     * @param string $limit advanced search limit
+     *
+     * @return array
+     */
+    public function topLimits($limit)
```

Comment:
I think it would be better to tie this functionality to the Search\Base\Options object rather than to the Config helper. This way, we can expand the feature to other search backends in the future if we need to.

Reply:
Yes, absolutely!